SPECS/safeclib/safeclib.spec

```diff
@@ -0,0 +1,72 @@
+Summary:        C11 Annex K functions
+Name:           libsafec
```

Comment:
Please always use the same value for the spec, its directory, and the `Name` tag inside the spec. Only exception is when one directory contains multiple spec files like `SPECS/coredns/coredns-[versionX].spec` for instance.